Debranding work and some README.md changes

Samisafool opened the pull request.

maximmasterr commented on cmake/UnitTest/test.rc:
```diff
@@ -14,7 +14,7 @@ BEGIN
                 BLOCK "000004b0"
                 BEGIN
-                        VALUE "CompanyName", "MultiMC Contributors"
+                        VALUE "CompanyName", "UltimMC Contributors"
```
Nope, most of code written by MultiMC so here should be MultiMC contributors

Samisafool replied:
Technically all the code in MultiMC is also part of UltimMC, so UltimMC contributors includes MultiMC contributors?

maximmasterr replied:
I guess no, people who contributed to MultiMC contributed to MultiMC bot UltimMC so they MultiMC contributors

Samisafool replied:
okay, will solve

maximmasterr commented on launcher/package/ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml:
```diff
@@ -6,7 +6,7 @@
     <description>
         <p>Overview</p>
-        <p>MultiMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
+        <p>UltimMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
```
Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by

maximmasterr commented on launcher/package/ubuntu/multimc/usr/share/man/man1/multimc.1:
```diff
@@ -28,20 +28,20 @@
 .SH "DESCRIPTION"
 .sp
-MultiMC is a custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once\&. It also allows you to easily install and remove mods by simply dragging and dropping\&. Here are the current features of MultiMC\&.
+UltimMC is a custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once\&. It also allows you to easily install and remove mods by simply dragging and dropping\&. Here are the current features of UltimMC\&.
```
Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by

maximmasterr commented on doc/multimc.1.txt:
```diff
@@ -5,31 +5,31 @@ MULTIMC(1)
 DESCRIPTION
 -----------
-MultiMC is a custom launcher for Minecraft that allows you to easily manage
+UltimMC is a custom launcher for Minecraft that allows you to easily manage
```
Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by

Samisafool commented:
Don't merge yet, will debrand more

Samisafool commented:
I found this one folder launcher/resources/multimc/ which I'm not sure is ok to change. lmk if you find out how to.

Samisafool commented:
Alright I'm done

Sebastian-byte commented:
Again, for debrand work, do it directly in the upstream launcher, doing it only here will only cause conflicts and make maintaining it more painful.

Samisafool commented:
It is not possible to debrand it upstream without creating similar conflicts, because you cannot use variables in the manpage or package description

Sebastian-byte commented:
> It is not possible to debrand it upstream without creating similar conflicts, because you cannot use variables in the manpage or package description or github issue templates

I don't really get what you mean by "conflicts" there, I'm talking about Git conflicts, and about that, in that case you should use the generic name, "Launcher"

Samisafool commented:
> I don't really get what you mean by "conflicts" there, I'm talking about Git conflicts, and about that, in that case you should use the generic name, "Launcher"

Did you see what I changed? There are URLs too, which I changed to this repo.

Sebastian-byte commented:
> Did you see what I changed? There are URLs too, which I changed to this repo.

Then consider making name changes in the upstream repository and then URLs changes in this repository later.
